.env.example

```diff
@@ -1,5 +1,6 @@
-NEXT_APP_PROJECT_ID="Appwrit Cloud project ID"
-NEXT_APP_DATABASE_ID="Appwrit Cloud database ID"
-NEXT_APP_USER_COLLECTION="Appwrit Cloud user collection ID"
-NEXT_APP_POSTS_COLLECTION="Appwrit Cloud post collection ID"
-NEXT_APP_BASE_URL="base url of the web app"
\ No newline at end of file
+NEXT_PUBLIC_PROJECT_ID="Appwrit Cloud project ID"
```

Comment:
Could you also fix the spelling of appwrite

Reply:
It's done.